x-pack/test/functional/services/transform/security_common.ts

```diff
@@ -21,7 +21,7 @@ export function TransformSecurityCommonProvider({ getService }: FtrProviderConte
     {
       name: 'transform_source',
       elasticsearch: {
-        indices: [{ names: ['*'], privileges: ['read', 'view_index_metadata'] }],
+        indices: [{ names: ['*'], privileges: ['read', 'view_index_metadata', 'manage'] }],
```

Comment:
*Temp addition to allow test power user to delete es indices

Reply:
See my comments in the test file. The `transform_dest` role allows to manage indices prefixed with `user-`. When that's changed in the test, this `transform_source` permission change should be reverted.